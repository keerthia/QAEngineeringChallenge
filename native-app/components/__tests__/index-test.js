
import * as React from 'react';
import renderer from 'react-test-renderer';
import ModalScreen from '../../app/modal';

it('Index page loads correctly', () => {
    const tree = renderer.create(<ModalScreen> Snapshot of ModalScreen</ModalScreen>).toJSON();
    console.log(tree);
    expect(tree).toMatchSnapshot();
}
);
