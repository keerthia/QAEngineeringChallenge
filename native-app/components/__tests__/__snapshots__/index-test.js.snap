// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page loads correctly 1`] = `
<View
  style={
    [
      {
        "backgroundColor": "#fff",
      },
      {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      },
    ]
  }
>
  <Text
    style={
      [
        {
          "color": "#000",
        },
        {
          "fontSize": 20,
          "fontWeight": "bold",
        },
      ]
    }
  >
    Modal
  </Text>
  <View
    style={
      [
        {
          "backgroundColor": "#eee",
        },
        {
          "height": 1,
          "marginVertical": 30,
          "width": "80%",
        },
      ]
    }
  />
</View>
`;
